packages/clerk-js/src/ui/elements/TagInput.tsx

```diff
@@ -137,15 +137,18 @@ export const TagInput = (props: TagInputProps) => {
           {tag}
         </TagPill>
       ))}
+
       <Input
         ref={inputRef}
         value={input}
         type='email'
+        data-testid='tag-input'
```

Comment:
How come we need this?

Reply:
What was the reason for this?

Reply:
@desiprisg We removed the placeholder that we were using to access it in the tests.

Reply:
Can you try using the `type` ?